acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -97,7 +102,7 @@ public void createKeyStoreYamlFileAt(
     final Path passwordFile = metadataFilePath.getParent().resolve(passwordFilename);
     createPasswordFile(passwordFile, password);
 
-    final Map<String, String> signingMetadata = new HashMap<>();
+    final Map<String, Serializable> signingMetadata = new HashMap<>();
```

Comment:
this can stay Map<String, String> (as String is derived from Serializable) (and elsewhere too)